synapse/storage/schema/full_schemas/README.txt

```diff
@@ -0,0 +1,14 @@
+Building full schema dumps
+==========================
+
+Postgres
```

Comment:
Can you add a comment to say that we need to wait for background updates to have completed?

Reply:
Done